### PR TITLE
New p:f:t tests

### PR DESCRIPTION
**Pull request by peterrum**
... tests to ensure that the order of pft and be the base triangulations are the same...

related to PR #9353 

FYI @agrayver

### CODE REVIEW

**Comment by agrayver** [comment]
Thanks for checking. I closed the issue.

File: tests/fullydistributed_grids/copy_distributed_tria_03.with_p4est=true.mpirun=5.output [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+
+
```
Comment (bangerth) [inline tests/fullydistributed_grids/copy_distributed_tria_03.with_p4est=true.mpirun=5.output]: Your test doesn't actually produce any output. Where do these empty lines come from?

File: tests/fullydistributed_grids/copy_serial_tria_08.mpirun=5.output [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+
+
+
```
Comment (bangerth) [inline tests/fullydistributed_grids/copy_serial_tria_08.mpirun=5.output]: same question

File: tests/fullydistributed_grids/copy_distributed_tria_03.with_p4est=true.mpirun=5.output (hunk 1 above)
Comment (peterrum) [reply tests/fullydistributed_grids/copy_distributed_tria_03.with_p4est=true.mpirun=5.output]: I think each process prints one empty line per default.

Comment (bangerth) [reply tests/fullydistributed_grids/copy_distributed_tria_03.with_p4est=true.mpirun=5.output]: I see.

Comment (kronbichler) [reply tests/fullydistributed_grids/copy_distributed_tria_03.with_p4est=true.mpirun=5.output]: If you don't need the output of every process, and it appears you don't, replace `MPILogInitAll all;` by `mpi_initlog();`

**Comment by peterrum** [comment]
I have updated the tests!

**Pull request merged**